ansible/vars/csm_packages.yml

```diff
@@ -55,3 +55,7 @@ compute_csm_sles_packages:
   - acpid
   - bos-reporter
   - cray-uai-util
+
+# List of packages to be installed/updated from Nexus on all storage nodes
+storage_csm_noos_packages:
```

Comment:
This name should follow the same pattern as the others
```suggestion
# List of packages to be installed/updated from Nexus on all storage Management NCNs
storage_mgmt_ncn_csm_sles_packages:
```

Reply:
done